### ISSUE TITLE
The `fixgalileo.sh` script can't seem to find the files to download anymore

### ISSUE THREAD

**Issue by acimmarusti** [majenkotech/IntelFixes#3]
It seems that the download mirror no longer has the necessary files.
Probably not fixable, but would really appreciate if you can share the files you may still have locally?

```
/fixgalileo.sh 
Arch: 
--2024-12-03 13:05:20--  https://downloadmirror.intel.com/24619/eng/galileo-toolchain-20150120-linux32.tar.bz2
Resolving downloadmirror.intel.com (downloadmirror.intel.com)... 3.163.189.107, 3.163.189.15, 3.163.189.102, ...
Connecting to downloadmirror.intel.com (downloadmirror.intel.com)|3.163.189.107|:443... connected.
HTTP request sent, awaiting response... 403 Forbidden
2024-12-03 13:05:21 ERROR 403: Forbidden.

tar (child): galileo-toolchain-20150120-linux32.tar.bz2: Cannot open: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now
./fixgalileo.sh: line 19: cd: /home/candres/.arduino15/packages/Intel/tools/i586-poky-linux-uclibc/1.6.2+1.0//i586: No such file or directory
sed: can't read install_script.sh: No such file or directory
sed: can't read relocate_sdk.py: No such file or directory
./fixgalileo.sh: line 22: ./install_script.sh: No such file or directory
```

**Comment by acimmarusti** [comment]
Managed to make it run further using this as an _alternative_ archive location:
http://retro.kab00m.ru/Abandonware%20Archives/Intel%20Support%20Archive/downloadmirror.intel.com/24619/eng/

Unfortunately this still failed, due to hard `python2` dependency

**Comment by majenkotech** [comment]
Yes, I just found the same site.

I've created a github "release" with those files attached so they can be downloaded from a more reliable site. Now just to adjust the script...

**Comment by majenkotech** [comment]
Fixed in 8f1d6ec
